RuleDocumentation/UseBOMForUnicodeEncodedFile.md

```diff
@@ -4,7 +4,8 @@
 
 ## Description
 
-For a file encoded with a format other than ASCII, ensure Byte Order Mark (BOM) is present to ensure that any application consuming this file can interpret it correctly.
+For a file encoded with a format other than ASCII, ensure Byte Order Mark (BOM) is present to ensure
```

Comment:
Is this still true or do we expect UTF8 (no BOM) now?

Reply:
The rule exists. I guess the question is whether it is or should be applied?

Reply:
@SteveL-MSFT the rule enforces the presence of a BOM.

A script encoded with UTF-8 (no BOM) will be misinterpreted by WinPS as CP-1252.

See https://docs.microsoft.com/powershell/scripting/dev-cross-plat/vscode/understanding-file-encoding

Reply:
(As it happens, that doc page itself has encoding issues around HTML escaping)

Reply:
Should we add the information about WinPS to the rule doc?

Reply:
@rjmholt FYI - just fixed the html encoding problem in the article. It will be live this afternoon.